haiku-apps/spybmessage/spybmessage-0.0~git.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="A command line tool to monitor BMessage events received by an application"
+DESCRIPTION="The spybmessage command is primarily useful as a debugging tool when writing graphical
+Haiku applications.  When run from a Terminal it will print out any BMessage events that the target
+application receives.
+
+This command comes from the Haiku source code repository but it is not included in normal builds of
+Haiku."
+HOMEPAGE="https://www.haiku-os.org/"
+COPYRIGHT="2008 François Revol (mmuman)"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="01f3ea96af191482839e6389621d45e56782618c"
+SOURCE_URI="https://raw.githubusercontent.com/haiku/haiku/$srcGitRev/src/bin/spybmessage.cpp#noarchive"
+CHECKSUM_SHA256="d9d5ed797d8b1031812d84c8e0bd1b0bc565d12c2f6c7c45bed6bf50cadf4724"
+
+ARCHITECTURES="all"
+SECONDARY_ARCHITECTURES="x86"
```

Comment:
Why do we need two versíons of this on x86? This is a command line tool, no library, so you can either specify "!x86_gcc2" or remove SECONDARY_ARCHITECTURES.

Reply:
It dynamically loads the executable and runs it from the app.  I believe it needs to match the compiler that the app was built with but I did not test it fully.